### PR TITLE
Istio Meetup China Report

### PR DESCRIPTION
**Pull request by irisdingbj**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/blog/2021/istiomeetups-china-report/index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+description: "中国首次Istio meetup圆满结束！"
+publishdate: 2021-07-15
+attribution: "Iris Ding (Intel), Maria Cruz (Google)"
+keywords: [community,meetup]
+---
+
+2021年7月10号Istio社区在中国的首场meetup在北京圆满结束。 这次活动得到了中国Istio开发者的热烈欢迎：有292人报名了本次活动，大约有140人现场参与了本次活动，在线观看直播的人数在高峰时段达到2000多人。 现场的与会者们也做了热烈的面对面的讨论，性能提升，数据平面的升级，跨集群管理是很多开发者关注的问题。 
+
+
+## 演讲资料
```
Comment (kebe7jun) [inline content/zh/blog/2021/istiomeetups-china-report/index.md:12]: Please add an id of this title.

Comment (irisdingbj) [reply content/zh/blog/2021/istiomeetups-china-report/index.md]: added :)

**Comment by irisdingbj** [comment]
@kebe7jun any clue for the current lint  failure? Thanks!

**Comment by kebe7jun** [comment]
> @kebe7jun any clue for the current lint failure? Thanks!

It seems a bug.
Your pdf file include a external url.
See:
https://github.com/istio/istio.io/blob/ed17aba8143a2c31238b5c051773e262dd558cbb/scripts/lint_site.sh#L74-L77

This script not bypass `*.pdf` files, you can try to add an `--exclude '*.pdf'` flag and retry it.

**Review by Xunzhuo — APPROVED** [review]

**Pull request merged**